src/nsrt_learning/option_learning.py

```diff
@@ -455,3 +458,10 @@ def update_segment_from_option_spec(self, segment: Segment,
         assert all(o.type == v.type for o, v in zip(objects, opt_vars))
         option = param_opt.ground(objects, params)
         segment.set_option(option)
+
+
+class _DirectBehaviorCloningOptionLearner(_BehaviorCloningOptionLearner):
```

Comment:
so future things (e.g. implicit BC) will differ only in the choice of regressor model class? not in data collection or anything?